content/guides/models/app/settings-page/billing-settings.md

```diff
@@ -41,5 +41,5 @@ To change the billing admin or assign the role to additional users:
 
 ## Invoices
 If you pay using a credit card, this section allows you to view monthly invoices.
-- For Enterprise accounts that pay via wire transfer, this section is blank.
+- For Enterprise accounts that pay via wire transfer, this section is blank. For questions, contact your account teams.
```

Comment:
Style guide question (I don't see it called out in Google Developer guide): Should we omit Latin words? At my old gig, we avoided terms like "via".

Reply:
Question: Do customers interact with multiple account teams? If nay, consider singular "...account team."

Reply:
"teams" is a typo bere. Good catch. 

I am OK with "via" if used sparingly. Somehow it reads better here than "using a wire transfer".